packages/framework/assets/js/admin/components/order.js

```diff
@@ -37,6 +37,9 @@ export default class OrderPreview {
                 interval: 200,
                 over: function () {
                     _this.$previewBox.show();
+                    _this.$previewBox.parents('table').removeClass('overflow-hidden').addClass('overflow-visible');
+                    _this.$previewBox.parents('table').parent().removeClass('overflow-hidden').addClass('overflow-visible');
```

Comment:
Ok, this is a very complicated process for me (parent, parent, ...). It would not be better to assign a `js-*` class to these elements and have on this place somethink like `_this.$previewBox.find(AllNodes)...`?

Reply:
I tried to recode this function, but I need to find js- classes in parents so I used $('body').find ... but I am not sure if this is good solution. Please can you check this?

Reply:
I think that is is great, thx.